### DISCUSSION TITLE
Creating new bones for SkyrimSE SMP

### DISCUSSION THREAD

**Discussion by Rasti-i** [BadDogSkyrim/PyNifly#232] [Q&A]
I'm wondering what steps to take to create new bones and have them properly export. What I tried was to create a couple new bones, parent them to the NPC Pelvis bone, create a couple new vertex groups for those and weight paint a piece of armor to them. But after exporting those bones and the weight painting just didn't transfer. How should I do this?

**Comment by BadDogSkyrim** [comment]
I didn't have a problem with this. Import the armor, edit the imported armature, add the bones. Add groups with those names to the armor. Weight paint mode, weight the groups. Export, you get the new bones. Moving the bones in nifskope deforms the armor as expected.

You might want to import the skeleton first and add the bones to that. Import the armor with the skeleton selected. But it depends on what you're doing.
